docs/sources/clients/promtail/stages/drop.md

```diff
@@ -0,0 +1,179 @@
+---
+title: drop
+---
+# `drop` stage
```

Comment:
Can you add a verb here? Or maybe change this to "Filter or drop logs"?

Reply:
for now I would like to leave this as is, which is consistent with how we title all our other stages.